### PR TITLE
[WIP] Add pde constrained example

### PR DESCRIPTION
**Pull request by JeremyFongSP**

### CODE REVIEW

File: docs/src/examples/pde_constrained.md [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+function burgers(u,p,t)
+    # Model parameters
+    a0, a1 = p
+    dx,Nx = xtrs #[1.0,3.0,0.125,100]
+    return 2.0*a0 .* u +  a1 .* d2dx(u, dx) #.*ddx(ddx(u, dx) ,dx)
+end
+
+# Testing Solver on linear PDE
+prob = ODEProblem(burgers,u0,tspan,p)
+sol = concrete_solve(prob,Tsit5(), dt=dt,saveat=t);
```
Comment (ChrisRackauckas) [inline docs/src/examples/pde_constrained.md:52]: this should just become `solve`

File: docs/src/examples/pde_constrained.md [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+
+plot(x, sol.u[1], lw=3, label="t0", size=(800,500))
+plot!(x, sol.u[end],lw=3, ls=:dash, label="tMax")
+
+ps  = [0.1, 0.2];   # Initial guess for model parameters
+# Defining Neural Net for HRT
+function predict(θ)  # Our 1-layer neural network
+    Array(concrete_solve(prob,Tsit5(),u0,θ,dt=dt,saveat=t))
```
Comment (ChrisRackauckas) [inline docs/src/examples/pde_constrained.md:60]: `solve`

Comment (ChrisRackauckas) [inline docs/src/examples/pde_constrained.md:60]:
```suggestion
    Array(solve(prob,Tsit5(),p=θ,dt=dt,saveat=t))
```

File: docs/src/examples/pde_constrained.md [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+function burgers(u,p,t)
+    # Model parameters
+    a0, a1 = p
+    dx,Nx = xtrs #[1.0,3.0,0.125,100]
+    return 2.0*a0 .* u +  a1 .* d2dx(u, dx) #.*ddx(ddx(u, dx) ,dx)
+end
+
+# Testing Solver on linear PDE
+prob = ODEProblem(burgers,u0,tspan,p)
+sol = concrete_solve(prob,Tsit5(), dt=dt,saveat=t);
```
Comment (ChrisRackauckas) [inline docs/src/examples/pde_constrained.md:52]:
```suggestion
sol = solve(prob,Tsit5(), dt=dt,saveat=t);
```

**Comment by JeremyFongSP** [comment]
I tried running it with `solve` instead of `concrete_solve` but it now gives me an error "try/catch is not supported"

**Comment by ChrisRackauckas** [comment]
You might need to update to the latest DiffEqBase

**Comment by JeremyFongSP** [comment]
I've updated it and it still gives me the same error. Any ideas?

**Comment by ChrisRackauckas** [comment]
Julia v1.4?

**Comment by JeremyFongSP** [comment]
Yep, 1.4.2

**Comment by ChrisRackauckas** [comment]
Try to directly add it to see what's blocking it

**Comment by JeremyFongSP** [comment]
Somehow, restarting the Julia REPL fixed my problem. Not sure why

**Comment by ChrisRackauckas** [comment]
If you update packages you need a new Julia session for it to replace what's running

**Pull request merged**